assignments/README.md

```diff
@@ -5,3 +5,12 @@ This space is for student assignments. It should not contain the solutions for o
 ## Propositional
 
 - [Cup of tea for grandpa](cup_of_tea/exercise.md)
```

Comment:
Was this line removed intentionally?

Reply:
I think it's actually fine -- the `-` symbol is just part of the markdown, and not an actual line removal.

Reply:
ha! I was browsing it on my phone and missed that detail.